### ISSUE TITLE
Elements stopped showing in iOS13

### ISSUE THREAD

**Issue by ne3Vubeki** [VitaliiBlagodir/cordova-plugin-datepicker#274]
![20190926_073034000_iOS](https://user-images.githubusercontent.com/3542667/65667916-66984380-e049-11e9-993d-603d642cf4b4.png)

**Comment by ne3Vubeki** [comment]
This is due to the installation of a dark theme, the font becomes white. You need to add the variable UIUserInterfaceStyle: Light to the file ...info.plist

**Comment by roxteddy** [comment]
Indeed. If I switch iPad to light theme, the text reappears. What's weird is my previous builds continue to work on iOS 13 dark theme. Only versions built after xcode update seems to be affected.

**Comment by ne3Vubeki** [comment]
All right. And you need to build with a new variable only in xcode 11

**Comment by roxteddy** [comment]
I was anxious to forget to modify this each time since I often reinstall platforms and plugins so I forked the plugin and added the parameter to the config XML.
I did a PR but since I can see the project has not moved for years, you can use my version if needed : https://github.com/roxteddy/cordova-plugin-datepicker

**Comment by loheander** [comment]
Wouldn't it be even better to switch the background color to dark grey if we detect the user is running dark theme?

**Comment by ne3Vubeki** [comment]
This is for someone like. I have a light theme in the application and I better get a white popup.

**Comment by amritk** [comment]
@ne3Vubeki your fix worked!

To use it in the config.xml:
```
<platform name="ios">
    <config-file parent="UIUserInterfaceStyle" platform="ios" target="**-Info.plist">
        <string>Light</string>
     </config-file>
</platform>
```
